rsocket/RSocket.h

```diff
@@ -24,8 +24,7 @@ class RSocket {
       std::shared_ptr<RSocketConnectionEvents> connectionEvents =
           std::shared_ptr<RSocketConnectionEvents>(),
       std::shared_ptr<ResumeManager> resumeManager = nullptr,
-      std::shared_ptr<ColdResumeHandler> coldResumeHandler =
-          std::shared_ptr<ColdResumeHandler>(),
+      std::shared_ptr<ColdResumeHandler> coldResumeHandler = nullptr,
```

Comment:
this change would break the RSocketClient::resume() (RSocketStateMachine::resume) method.

Reply:
Actually I only need to update the `createClientFromConnection`, for Thrift integration purposes. Let me remove the remaining updates in this file.